qiskit/circuit/measure.py

```diff
@@ -59,3 +60,80 @@ def measure(self, qubit, cbit):
 
 
 QuantumCircuit.measure = measure
+
+
+class PauliMeasure(Instruction):
```

Comment:
should we call this `MeasurePauli` to emphasize the measurement part and it appears next to regular Measure in the list of instructions/circuit methods.

Reply:
agreed, done